### DISCUSSION TITLE
Color Code status

### DISCUSSION THREAD

**Discussion by stephensanwo** [community/community#12579] [Projects and Issues] [answered]
It's difficult to the tell at a glance when a status is in progress or completed, especially when you have a really long list. Please provide an option to color-code status.

**Comment by z64555** [comment]
I think the status color coding should have a similar UI to that of issue tags.

**Comment by j0siepy** [comment, marked as answer]
Hi folks!! Thank you so much for your patience - we've made this possible! Let us know if you have any future feedback for us ❤️

https://user-images.githubusercontent.com/98360703/217907904-3b34ea95-7638-4eff-b692-131275d1c19a.mov
